stylesheets/widgets/mz-search-pagination-dd.less

```diff
@@ -0,0 +1,82 @@
+.mz-dd-search {
```

Comment:
Your CSS naming convention is not consistent. Some use dashes, others camel case, and others underscores. You should pick one style. Most CSS devs use the lowercase dash format but camel case is also OK. Just pick one.

I would also normally recommend to encapsulate all your internal CSS classes inside a parent class like `.mozu-dropdown` which will keep your CSS rules cleaner.

Reply:
Done, Thanks